demos/text_detection_demo/cpp/main.cpp

```diff
@@ -210,12 +213,16 @@ int main(int argc, char *argv[]) {
                 double conf = 1.0;
                 if (text_recognition.is_initialized()) {
                     auto blobs = text_recognition.Infer(cropped_text);
-                    auto output_shape = blobs.begin()->second->getTensorDesc().getDims();
+                    auto out_blob = blobs.begin()->second;
+                    if (FLAGS_tr_o_blb_nm != "")
+                        out_blob = blobs[FLAGS_tr_o_blb_nm];
+                    auto output_shape = out_blob->getTensorDesc().getDims();
+
                     if (output_shape[2] != kAlphabet.length()) {
```

Comment:
Please check `output_shape.size()` is at least 3. As I understand one of the outputs is of rank two and this results in reading garbage if wrong name is given.

Reply:
Added check